registration/templates/registration/login_form.html

```diff
@@ -1,24 +0,0 @@
-<form id="login-form" name="{{ form.form_name }}" action="" method="post" novalidate>
```

Comment:
We'll keep the login form to log in Ocim management console.

Reply:
Yeah, I have unremoved it :+1: